WordPress/gutenberg
# WordCount: Add types

Part of #18838.

Add types to `wordcount` package.

**Testing Instructions**

0. Run `npm run build:package-types` –– shouldn't be any errors.
1. Try it in the UI – open the sample post and it should say something around 717 words. And start a new post and it should count words properly.

I am quite new to TypeScript and adding types to Gutenberg, so for the sake of learning, bring all of your nitpicking :-)
